shuffle/app/components/polls/responses/StatementSort.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+
+import type { FC } from "react";
+import React, { useCallback } from "react";
+
+import { usePathname, useRouter } from "next/navigation";
+
+import type { SortKey } from "@/lib/pollResults/constants";
+import { SORT_PARAM, sortOptions } from "@/lib/pollResults/constants";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/shadcn/select";
+
+type Props = {
+  value?: SortKey;
+};
+
+export const StatementSort: FC<Props> = ({ value = "consensus" }) => {
+  const router = useRouter();
+  const pathname = usePathname();
+
+  const handleChange = useCallback(
```

Comment:
Im not sure its worth wrapping simple things like this in useCallback. It might be worse performance wise than just having the function itself.

Reply:
You're spot on, I removed it.